src/main/java/com/fasterxml/jackson/datatype/joda/ser/DaysSerializer.java

```diff
@@ -0,0 +1,28 @@
+package com.fasterxml.jackson.datatype.joda.ser;
+
+import java.io.IOException;
+
+import org.joda.time.Days;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+/**
+ * Serializer for Joda {@link Days} class.
+ *
+ * @since 2.19
+ */
+public class DaysSerializer
+    extends JodaSerializerBase<Days>
+{
+    private static final long serialVersionUID = 1L;
+
+    public DaysSerializer() {
+        super(Days.class);
+    }
+
+    @Override
+    public void serialize(Days value, JsonGenerator gen, SerializerProvider provider) throws IOException {
+        gen.writeNumber(value.getDays());
```

Comment:
This works for now. In future we could also add support for "Stringified" numbers via `@JsonFormat.shape` (but would be overkill for now I think... plus Joda module may be getting less use over time).
So just mentioning for future expansion, no need to add yet.